nutrionator.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+import numpy.typing as npt
+
+'''
+Chciałem zrobić klasę z interfejsem podobnym do klasyfikatorów z scikit learna.
+Oczywiście udało się bardziej mniej niż więcej, ale oceńcie sami.
+
+Nutrinator to nowy wynalazek dr Dundersztyca. 
+Dzięki niemu będzie w stanie się dobrze odżywiać,
+wykształci dobrze zbudowaną sylwetkę
+i będzie mógł walczyć z PP Panem Dziobakiem pięść w pięść ... czy tam łapę ... czy co tam mają dziobaki
+
+Dundersztyc będzie niepokonany - nie ma tutaj nawet przycisku autodestrukcji! 
+'''
+class Nutrinator:
+
+    def __init__(self, N: dict, gamma=None):
+        self.N = N
+        self.gamma = gamma if gamma else np.array([1, 1, 1, 1])
+        self.n_b = None
+        self.n_s = None
+
+    def fit(self, n_b, n_s=None):
+        self.n_b = n_b
+        self.n_s = n_s if n_s is not None else np.zeros(n_b.shape)
+
+    def compute(self, A):
+        return np.abs(self.gamma.T @ self.__d_n(A))
+
+    # strasznie nie podoba mi się, że musiałem dodać tutaj pętle z pythona
+    # Jeżeli macie pomysł na to, jak to przyśpieszyć, to dajcie znać,
```

Comment:
Możemy wstawić [`np.vcetorize`](https://numpy.org/doc/stable/reference/generated/numpy.vectorize.html), co prawda [dla dużych macierzy nic nie robi](https://stackoverflow.com/questions/35215161/most-efficient-way-to-map-function-over-numpy-array), ale w tym przypadku mamy raczej małe macierze

Reply:
Możemy na razie podmienić na `np.vectorize` i zostawić, a jak komuś wpadnie jakiś ***big brain time*** to wtedy podmienić

Reply:
Bawiłem się tym, ale jakoś nie wyszły z tego żadne ciekawe wyniki. Mam wrażenie, że `vectorize` działa tylko, jeżeli mamy funkcję, której wejściem jest skalar i wyjściem też skalar. Wiele funkcji tutaj dodaje albo redukuje wymiar i chyba nie da rady zastosować `vectorize` do nich. Proszę mnie poprawić, jeżeli się mylę.
W każdym razie - nie chcę tego implementować. @Thematiq oceń, czy to, co jest, póki co wystarczy